source/infrastructure/lib/core-scheduler.ts

```diff
@@ -48,9 +47,50 @@ export class CoreScheduler {
 
   constructor(scope: Stack, props: InstanceSchedulerLambdaProps) {
 
-    const lambdaToDynamoDbConstruct = new LambdaToDynamoDB(scope, 'instance-scheduler-lambda', extractLambdaToDynamoPropsFrom(props))
+    this.lambdaFunction = new python.PythonFunction(scope, "scheduler-lambda",{
+      functionName: Aws.STACK_NAME + '-InstanceSchedulerMain',
+      description: 'EC2 and RDS instance scheduler, version ' + props.solutionVersion,
+      entry: "../app",
+      index: "main.py",
+      handler: 'lambda_handler',
+      runtime: lambda.Runtime.PYTHON_3_9,
+      role: props.schedulerRole,
+      memorySize: props.memorySize,
+      timeout: cdk.Duration.seconds(300),
+      environment: props.environment,
+      tracing: lambda.Tracing.ACTIVE,
+
+      bundling: {
+        commandHooks: {
+          beforeBundling(inputDir: string, outputDir: string): string[] {
+             return [`pip install --target ${outputDir} -e ${inputDir}`]
```

Comment:
hmm, this may be the origin of the "editable" files I had been seeing. I had assumed the -e was required for specifying the input dir but I may have been mistaken. I'll try removing it and see what it does

Reply:
yes, that's exactly what that was. good catch.

Reply:
Apparently losing this flag breaks the docker build of the lambda... Have re-added it and re-opened the story to investigate it more deeply